src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -5525,6 +5525,13 @@ public boolean isThisLatestReleasedVersion() {
 
     }
 
+    /**
+     * JSON-LD is provided via the normal ExporterService. In favour for the more generic implementation this method is
+     * deprecated.
+     * @see  #getExporters()
+     * @return The Dateset encoded as JSON-LD
+     */
+    @Deprecated
```

Comment:
This method is used to add the schema.org metadata to the header of the dataset page. I don't see any place that you've replaced that functionality so this would need to not be deprecated then?